metriq/models/task.py

```diff
@@ -57,9 +57,16 @@ class TaskCreateRequest(TeaClientModel):
     """
 
     name: str
+    fullname: Optional[str]
     description: str = ""
+    isHideChart: Optional[str]
+    createdAt: Optional[str]
+    updatedAt: Optional[str]
+    userId: Optional[str]
     area: Optional[str] = None
     parent_task: Optional[str] = None
+    childTasks: Optional[str]
+    submissions: Optional[str]
```

Comment:
Karen, this is a lot better! However, `childTasks` and `submissions` are lists of sub-models that can be removed, for now. Thank you!